### DISCUSSION TITLE
Is there an API to trigger emails being sent, the content, recipient etc?

### DISCUSSION THREAD

**Discussion by oscadev** [pocketbase/pocketbase#776] [Q&A] [answered]
I have the smtp mail server setup correctly so that using "send test email" works. However, is there a way to trigger an email being sent through the /api/ ? I have a contact form that I'd like to hookup to this mailer. If there isnt an API way, I will just use nodemailer for that particular task until its implemented later. Thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
>  is there a way to trigger an email being sent through the /api/

Not directly. If you use PocketBase as framework you can attach to one of the [event hooks](https://pocketbase.io/docs/event-hooks/) and access the mailer service to send your own custom emails:

```go
client := app.NewMailClient()

err := client.Send(
    mail.Address{Address: "from@example.com"},
    mail.Address{Address: "to@example.com"},
    "YOUR_SUBJECT...",
    "YOUR_HTML_BODY...",
    nil, // attachments
)
```

**Reply by mac501pl** [reply]
Would you consider adding sending custom emails to the JS SDK?
